modules/protect/blocked-login-page.php

```diff
@@ -236,7 +239,7 @@ function send_recovery_email() {
 		$user = get_user_by( 'email', trim( $email ) );
 
 		if ( ! $user ) {
-			return new WP_Error( 'invalid_user', __( 'Oops, could not find a user with that email address.', 'jetpack' ) );
+			return new WP_Error( 'invalid_user', __( 'Oops, we couldn’t find a user with that email. Please try again!', 'jetpack' ) );
```

Comment:
That `’` character may cause some confusion down the road. Should we just use `'`?